lecture-notes/2-vcs.md

```diff
@@ -150,14 +150,14 @@ current state of things is. It shows you a description of what's going on, as
 well as some suggested things to do next. When you run it in a new repository,
 it helpfully suggests that you make and *track* some files:
 
-```
+```console
```

Comment:
nit: `?prompt=$`

Reply:
This block highlights fine without it. Do you just want to include it everywhere for consistency?